depends/config.guess

```diff
@@ -141,7 +141,7 @@ set_cc_for_build() {
 # This is needed to find uname on a Pyramid OSx when run in the BSD universe.
 # (ghazi@noc.rutgers.edu 1994-08-24)
 if test -f /.attbin/uname ; then
-	PATH=$PATH:/.attbin ; export PATH
```

Comment:
Is this actually needed? Seems like a completely arcane check and I'm not sure how this would be tokenized wrong.

Reply:
This is also an upstream file, so shouldn't be modified here. If we do want to change it, we should first pull the latest upstream version, and also send a patch upstream.

Reply:
It's been updated as recently as April of this year. Gettin git fixed upstream would be ideal, but surely that shouldn't get in the way of fixing the issue locally?

Reply:
Is this actually fixing anything though?